### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by melaniesigrid**
## Hello there :alien:

Our project should be finished! :boom:
Thank you for reviewing it. :cat: Any suggestions to improve will be greatly appreciated as we love new opportunities to learn! :fire:

:page_with_curl: On this PR: 
- We restructured our page to include each part of the functionality in different sections. :triangular_ruler:
- We added a navbar, a footer and some cute decorations. :sparkles:
- We followed the SPA model, using JavaScript to toggle between the different sections in our page. :wrench:

Thanks again, we hope you enjoy it!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,27 +13,66 @@
...
+    <section class="contact d-none" id="contact">
+      <h2>Contact information</h2>
+      <p>Do you have any questions or you just want to say "Hello"?<br>You can reach out to us!</p>
+      <ul>
+        <li>Andrew's email: andresf.2117@gmail.com</li>
+        <li>Melanie's email: melaniesigrid@protonmail.com</li>
+        <li>Andrew's Twitter: <a href="https://twitter.com/johnftitor" target="_blank" rel="noopener noreferrer">@johnftitor</a></li>
+        <li>Melanie's Twitter: <a href="https://twitter.com/SiriMoon11" target="_blank" rel="noopener noreferrer">@SiriMoon11</a></li>
+      </ul>
+    </section>
```
Comment (bimbolabuari) [inline index.html]: Kindly make the navigation link appear in the contact section so as to navigate to other links 👍 

> image

> image

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi Team, 

You have done a good job so far 🚀  

## Highlights 🎉 

- Nice UI design
- Functional Navigation links


There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: index.html (hunk 1 above)
Comment (JohnFTitor) [reply index.html]: Hello! @bimbolabuari 

There seems to be a strange issue. On our screen, the navigation is in the contact section as shown in the screenshot below. Can I ask you to check again?
Navigaytion

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (JohnFTitor) [reply index.html]: After checking again the project, we realized we had something wrong with the responsiveness, thus creating this issue on your screen. We fixed it so it's working now. Thanks for letting us know! 😸

**Pull request merged**